common/changes/@microsoft/rush/feat-pnpm-patch_2022-07-27-03-39.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "rush-pnpm accepts patch and patch-commit",
+      "type": "none"
+    }
+  ],
+  "packageName": "@microsoft/rush"
+}
```

Comment:
FYI when I tried the test repo https://github.com/microsoft/rushstack/pull/3554 , the `rush install` failed because your **main** branch is using **pnpm-config.json** whereas **rush.json** specifies an old release that doesn't support that config file.